jasmin_services/templates/jasmin_services/service_details.html

```diff
@@ -258,6 +258,44 @@ <h3 class="card-title">Current Requests</h3>
                             {% endfor %}
                         </table>
                     </div>
+                    
+                {% endif %}
+                {% if grants %}
+                    <div class="card">
+                        <div class="card-header">
+                            <h3 class="card-title">Managers & Deputies</h3>
+                        </div>
+                        <table class="table table-hover access-table">
+                            <thead>
+                                <tr>
+                                    <th></th>
+                                    <th>Role</th>
+                                    <th>Name</th>
+                                    <th></th>
+                                </tr>
+                            </thead>
+                            {% for manager in managers%}
+                                <tbody>
```

Comment:
You don't need to start a new <tbody> for each row- you can just have one around all the rows, and if you do this it looks a bit nicer: 
![image](https://user-images.githubusercontent.com/33454849/193547943-f50bb9ff-d195-44f2-8d71-1f999027f8a2.png)